### DISCUSSION TITLE
Welcome to vue-timeline-chart Discussions!

### DISCUSSION THREAD

**Discussion by laurens94** [laurens94/vue-timeline-chart#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stevemandl** [comment]
I'm trying to use this for the first time and I'm having trouble getting scrolling to work.
Here is a stackblitz to show what I have so far. Can you help me get this working? I'll be happy to leave up a working sample in stackblitz once it gets working.
https://stackblitz.com/~/github.com/stevemandl/vue-timeline-chart-test

**Reply by laurens94** [reply]
Hey! 
Do you mean scrolling horizontally through time? For that you can try Shift+Scrolling (or a trackpad/mouse with horizontal scrolling). 
Let me know if it works!

**Reply by stevemandl** [reply]
Thanks for this sweet component, and thanks also for the answer.
Yes, shift+scroll was it. I also discovered ctrl+scroll zooms the view. My mouse horizontal scroll does nothing. I was expecting the time range to zoom with the scroll wheel when the mouse was over the component, and I did not think to use the ctrl and shift keys.
